src/bitmessagemain.py

```diff
@@ -238,8 +255,28 @@ def start(self):
             elif opt in ("-c", "--curses"):
                 state.curses = True
             elif opt in ("-t", "--test"):
-                state.testmode = daemon = True
+                state.testmode = True
+                if os.path.isfile(os.path.join(
+                        state.appdata, 'unittest.lock')):
+                    daemon = True
+                else:
+                    from tests import core as test_core
```

Comment:
Please use a comment to suppress the pylint warning, something like

    from tests import core as test_core # pylint: disable=relative-import